nginx/nginx.conf

```diff
@@ -0,0 +1,18 @@
+upstream django_app {
```

Comment:
I would prefer to have the config in the gamification style
There is no need to use upstream here
it's like creating a function with a single like that call another function.